tensorflow/core/kernels/mkl_fused_batch_norm_op.cc

```diff
@@ -281,6 +386,8 @@ struct MklBatchNormBwdParams {
   int depth;
   float eps;
   bool training;
+
+#ifndef ENABLE_MKL_DNN_V1
```

Comment:
Typo?
```suggestion
#ifndef ENABLE_MKLDNN_V1
```
I see this in several other places. Please fix them too.

Reply:
Done